versions/2016-10-31.md

```diff
@@ -429,6 +429,7 @@ Property Name | Type | Description
 Topic | `string` | **Required.** Topic ARN.
 Region | `string` | Region.
 FilterPolicy | [Amazon SNS filter policy](https://docs.aws.amazon.com/sns/latest/dg/message-filtering.html) | Policy assigned to the topic subscription in order to receive only a subset of the messages.
+SqsSubscription | `boolean` | Indicates whether SQS option is enabled.
```

Comment:
I'm sorry but I'm non-native english speaker so could anyone provide appropriate description for `SqsSubscription`?

Reply:
Sure. I would say:

```
SqsSubscription | `boolean` | Set to `true` to enable batching SNS topic notifications in an SQS queue.
```

Reply:
Thank you for giving me a hand!
I made suggested change in e7e2dd8.